Dockerfile

```diff
@@ -44,3 +44,5 @@ VOLUME ["/data/logs", "/data/cache", "/data/cachedomains", "/var/www"]
 
 EXPOSE 80 443
 WORKDIR /scripts
+
+HEALTHCHECK CMD curl --fail http://127.0.0.1/lancache-heartbeat  || exit 1
```

Comment:
Nice improvement, i've been meaning to do this forever!
We should probably define some start's and intervals. How about?
`HEALTHCHECK --interval=1m --timeout=1s --start-period=120s CMD curl -f http://localhost/ || exit 1`